.gitignore

```diff
@@ -104,3 +104,4 @@ Thumbs.db
 /backup-routes.py
 /worke.py
 /branches/
+/shared/
```

Comment:
what's this folder used for?

Reply:
a common place to share data that you want saved over updates ... i don't want to check my keys in again :P

Reply:
taking this back out ... makes sense as a user I'd want my keys in the data directory